collectors/health.go

```diff
@@ -615,6 +645,36 @@ func (c *ClusterHealthCollector) collect() error {
 	}
 	c.RemappedPGs.Set(remappedPGs)
 
+	scrubbingPGs     := float64(0)
+	deepScrubbingPGs := float64(0)
+	for _, pg := range stats.PGMap.PGsByState {
+		matched, err := regexp.MatchString(`scrubbing`, pg.StateName)
```

Comment:
Any objection to have this transformed into [strings.Contains()](https://golang.org/pkg/strings/#Contains), as that's a canonical version of this? :)